app/routes/schemePaymentsByYear.ts

```diff
@@ -9,7 +9,10 @@ module.exports = [
     options: {
       auth: false,
       handler: async (_request: Request, h: ResponseToolkit): Promise<ResponseObject> => {
-        return h.view('scheme-payments-by-year/index', await schemePaymentsByYearModel())
+        return h.view('scheme-payments-by-year/index', {
+          ...(await schemePaymentsByYearModel()),
+          relatedContentData: getRelatedContentLinks('scheme-payments-by-year'),
```

Comment:
Similarly, please put the relatedContentData in schemePaymentsByYearModel.

Reply:
Thanks FYC, Done!